src/Imdb/Title.php

```diff
@@ -1607,7 +1607,7 @@ public function episodes() {
       if (!$this->seasons()) {
         $ser = $this->get_episode_details();
         if (isset($ser['imdbid'])) {
-          $show = new Title($ser['imdbid'], $this->config);
```

Comment:
Can you undo this change please, this might solve one problem but it'll create another